torch/quantization/fx/prepare.py

```diff
@@ -160,7 +166,10 @@ def insert_observer(
     if model_device:
         observer.to(model_device)
     # add observer module as attribute
-    prefix = node.name + '_activation_post_process_'
+    if isinstance(observer, _InputEqualizationObserver) or isinstance(observer, _WeightEqualizationObserver):
```

Comment:
nit: maybe put this check in a function like https://github.com/pytorch/pytorch/blob/master/torch/quantization/quantize.py#L23